src/D3MMom.sol

```diff
@@ -16,16 +16,16 @@
 
 pragma solidity 0.6.12;
 
-interface FileLike {
-    function file(bytes32 what, uint256 data) external;
+interface DisableLike {
+    function disable() external;
 }
 
 interface AuthorityLike {
     function canCall(address src, address dst, bytes4 sig) external view returns (bool);
 }
 
 // Bypass governance delay to disable a direct deposit module
-contract DirectDepositMom {
+contract D3MMom {
     address public owner;
     address public authority;
 
```

Comment:
On lines 32 and 33, we can remove oldOwner and oldAuthority, respectively, from the events.

Reply:
e6ba624